transaction-flow.md

```diff
@@ -123,14 +127,13 @@ A full example transaction creation request with sender creation from EUR to NGN
         }
       }
     ],
-    "metadata": {
-      "sendRef": "TRANSACTION-1234"
-    }
+    "metadata": {},
+    "external_id": "806ec63a-a5a7-43cc-9d75-1ee74fbcc026"
   }
 }
 ```
 
-Note that in the `metadata` section if you use `sendRef` to send us your internal reference, that value will be displayed in your transaction reports.
+Note that the `external_id` field is optional. This internal reference value will be displayed in your transaction reports.
```

Comment:
I would add much more details about this feature here, including what it means, what it does, how it can be used to ease duplicate checking, etc. See below on more comments for example on the available options

Reply:
I've added a more detailed External ID section that should cover this. Does it need more detail on the options as with the sender section?